File: 백준/Bronze/10798. 세로읽기/세로읽기.java
import java.io.*;
import java.rmi.activation.ActivationGroup_Stub;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        Character[][] characters = new Character[5][15];

        int max = 0;

        for (int i = 0; i < 5; i++) {
            String str = br.readLine();

            max = Math.max(max, str.length());

            for (int j = 0; j < str.length(); j++) {
                characters[i][j] = str.charAt(j);
            }
        }

        StringBuilder sb = new StringBuilder();

        for (int i = 0; i < max; i++) {
            for (int j = 0; j < 5; j++) {
                if (characters[j][i] != null) sb.append(characters[j][i]);
            }
        }

        System.out.println(sb);
    }
}

File: 백준/Bronze/10798. 세로읽기/세로읽기.c
#include <stdio.h>

int main() {
	char characters[5][16] = { 0 };

	for (int i = 0; i < 5; i++) {
		scanf("%s", characters[i]);
	}

	for (int i = 0; i < 15; i++) {
		for (int j = 0; j < 5; j++) {
			if (characters[j][i] != '\0') printf("%c", characters[j][i]);
		}
	}
}